src/SprykerEco/Zed/Payone/Persistence/PayoneRepositoryInterface.php

```diff
@@ -49,7 +49,7 @@ public function getPaymentDetail(int $idOrder): PaymentDetailTransfer;
     /**
      * Gets payment logs (both api and transaction status) for specific orders in chronological order.
      *
-     * @param \ArrayObject|\Generated\Shared\Transfer\OrderTransfer[] $orders
+     * @param \Generated\Shared\Transfer\OrderTransfer[]|\ArrayObject $orders
```

Comment:
```suggestion
     * @param \ArrayObject<array-key, \Generated\Shared\Transfer\OrderTransfer[]> $orderTransfers
```

Reply:
If the code sniffer doesnt take care of this, we dont have to at this point.
Future versions will do that then.

Reply:
Fixed anyway.